gtfs_realtime_translators/factories/factories.py

```diff
@@ -70,6 +71,8 @@ def create(*args, **kwargs):
         stop_time_update = gtfs_realtime.TripUpdate.StopTimeUpdate(arrival=arrival,
                                                                    departure=departure,
                                                                    stop_id=stop_id)
+        
+        vehicle_descriptor = gtfs_realtime.VehicleDescriptor(id=run_number)
```

Comment:
Do we need to include an `id` field? If not, let's only include the `run_number` extension field.

Reply:
Is it possible to only include our optional `run_number` field in the VehicleDescriptor and not include the `id` field though - can we instantiate an empty `gtfs_realtime.VehicleDescriptor()`, and then populate the `run_number` field if it exists?